pythonforandroid/bootstraps/webview/build/templates/AndroidManifest.tmpl.xml

```diff
@@ -77,6 +77,7 @@
             {%- if args.intent_filters -%}
             {{- args.intent_filters -}}
             {%- endif -%}
+            <meta-data android:name="WindowManagerPreference:SuppressWindowControlNavigationButton" android:value="true" />
```

Comment:
This looks good but it will only work for our app. A better way to do it is by adding an argument that we can pass to the template and then set this meta-data conditionally. Here is one example of a boolean argument from `pythonforandroid/bootstraps/common/build/build.py`:

```
    ap.add_argument('--wakelock', dest='wakelock', action='store_true',
                    help=('Indicate if the application needs the device '
                          'to stay on'))
```

And then it's used in the template like this:
```
    {% if args.wakelock %}
    <uses-permission android:name="android.permission.WAKE_LOCK" />
    {% endif %}
```